apps/web/src/modules/proposal/components/ProposalNavigation.tsx

```diff
@@ -66,7 +75,25 @@ export const ProposalNavigation: React.FC<ProposalNavigationProps> = ({ handleBa
             )}
           </Flex>
         </Box>
+        {transactionType ? (
+          <Box>
+            <Flex>
+              <Button mr="x6" variant="secondary" onClick={() => setQueueModalOpen(true)}>
+                {`${transactions.length} transactions queued`}
+              </Button>
+              <OptionalLink
+                enabled={!!transactions.length}
+                href={`/dao/${token}/proposal/review`}
+              >
+                <Button disabled={!transactions.length}>Continue</Button>
+              </OptionalLink>
+            </Flex>
+          </Box>
+        ) : null}
       </Flex>
+      <AnimatedModal close={() => setQueueModalOpen(false)} open={queueModalOpen}>
```

Comment:
non-blocking comment and a ux question (so feel free not to address) - maybe we could consider adding a 'x' close button on the animated modal itself so then there wouldn't need to be an explicit close button within the modal content?

Reply:
good point will discuss with @isaaczaak

Reply:
thanks @nadiakiamilev for the suggestion! we've opened an issue for this #196